controllers/nginx/pkg/template/template.go

```diff
@@ -434,12 +437,6 @@ func buildRateLimit(input interface{}) []string {
 		limits = append(limits, limit)
 	}
 
-	if loc.RateLimit.RPM.Limit > 0 {
```

Comment:
These lines of code are a duplicate.

Reply:
This is not duplicated RPM != RPS

Reply:
Just noticed! Thanks @aledbf I just pushed the fix.